sjb/actions/clonerefs.py

```diff
@@ -57,16 +59,23 @@ def generate_parameters(self):
             _PARAMETER_TEMPLATE.render(name='CLONEREFS_ARGS', decsription='Pull request head SHA.'),
         ]
 
-    def generate_build_steps(self):
-        return [render_task(
-            title="FORWARD GCS CREDENTIALS TO REMOTE HOST",
-            command="scp -F ./.config/origin-ci-tool/inventory/.ssh_config /var/lib/jenkins/.config/gcloud/gcs-publisher-credentials.json openshiftdevel:/data/credentials.json"
-        )] + ForwardParametersAction(
+    def generate_build_steps(self, output_format):
```

Comment:
Would be cleaner IMO if you made a new interface for all steps `generate_shell_steps()` and plumbed it through appropriately

Reply:
Done. Now `output_format` is Action var, passed to `render_task`.

Should `render_task` become a part of Action, so that `output_format` would stay inside `Action`?